package.json

```diff
@@ -454,10 +454,6 @@
           "command": "confluent.schemaViewer.viewLocally",
           "when": "false"
         },
-        {
-          "command": "confluent.schemaViewer.uploadVersion",
-          "when": "false"
```

Comment:
We probably still want `confluent.schemas.upload` excluded from the command palette, right?

Reply:
Thanks, good eyes.